CASMNET-1242 (#1231)
Cray-HPE/docs-csm: operations/network/management_network/load_saved_switch_config.md

# Load Saved Switch Configuration

> This procedures shows how to switch between already saved switch configurations.

Reviewer: ```suggestion
This procedure shows how to switch between already saved switch configurations.
```

To save switch configurations, refer to the [Configuration Management](config_management.md) procedure.

This procedure is intended for internal use only.  It's used to quickly switch between configurations that are already loaded on the switches.

This procedure needs to be done on all mgmt switches.

-  Spine switches will have three total configuration files/checkpoints.
    -  1.2 fresh install
    -  1.2 upgrade
    -  1.0
  
-  Leaf-BMC switches will have have two configuration files/checkpoints.
    - 1.2
    - 1.0
  
### Aruba

1. View the checkpoints. 
   

>     Ensure that the proper checkpoints exist. In this example we will be using `CSM1_0` and `CSM1_2`

Reviewer: ```suggestion
    Ensure that the proper checkpoints exist. `CSM1_0` and `CSM1_2` are used in this example.
```

    Example:

    ```
    sw-spine-001# show checkpoint | include CSM
    CSM1_2_FRESH_INSTALL_CANU_1_3_2                     latest      User    2022-04-01T20:11:57Z  GL.10.09.0010
    CSM1_2_UPGRADE_CANU_1_3_2                           checkpoint  User    2022-04-01T18:57:06Z  GL.10.09.0010
    CSM1_0_CANU_1_2_4                                   checkpoint  User    2022-03-15T21:37:11Z  GL.10.09.0010
    ```

2. Rollback to desired checkpoint.

    ```
    sw-spine-001# checkpoint rollback CSM1_2_UPGRADE_CANU_1_3_2    
    ```

### Dell

1. View the configuration files.
   

>     Ensure that the proper config files exist. In this example we will be using `CSM1_0` and `CSM1_2`

Reviewer: ```suggestion
    Ensure that the proper config files exist. `CSM1_0` and `CSM1_2` are used in this example.
```

1. View the configuration files.
   

>     Ensure that the proper checkpoints exist. In this example we will be using `CSM1_0` and `CSM1_2`

Reviewer: ```suggestion
    Ensure that the proper checkpoints exist. `CSM1_0` and `CSM1_2` are used in this example.
```
